library/koji_tag_inheritance.py

```diff
@@ -211,21 +224,23 @@ def remove_tag_inheritance(session, child_tag, parent_tag, priority,
     """
     result = {'changed': False, 'stdout_lines': []}
     current_inheritance = session.getInheritanceData(child_tag)
-    found_rule = {}
+    found_rules = []
```

Comment:
I'm not sure about this change. Have you ever seen this happen in a Koji environment anywhere?

This makes the code harder to understand to me, like why would we be searching for multiple "rules" if the point of this method is to only ever delete one?

Reply:
I thought it would be better to know if an unexpected scenario appeared—perhaps because of a bug (or change in behavior) in Koji or another component—rather than implicitly choose potentially bogus data. However, assuming that the Koji data will be valid (i.e., no two parents of a tag will share a name or a priority) is how the rest of the module is structured, so I've removed the check.